MAINTAINING.md

```diff
@@ -36,12 +36,14 @@ the overall validity of the PR.
     Is the problem that the PR intends to solve clearly stated?
 2.  Is the solution presented in the PR as simple as possible to satisfy the
     requirements, but no simpler?
-3.  Is the PR functionally atomic? Does it address a single issue or should
-    it be split into multiple individual PRs?
+3.  Is the PR a manageable size for you to handle? As a general rule of thumb,
```

Comment:
While I agree that the requirement to be "atomic" might lead to single-line breakout PR's for three line changes, the rule of thumb should still be "as small as possible".  Where does that full day come from?

PR's are usually too large and rarely too small, thus exaggerating towards small might be beneficial. Also, it is much easier to cherry-pick more commits or join two PR's than to split them.

Reply:
Ok thanks. Re the PR being runnable - would this mean that it doesn't, on its own, leave some area of the code in a broken state? For a new feature, we would say that the feature has to work after that one PR gets merged, and for a bugfix, the bug needs to be fixed in that PR and that PR only? And subject to these conditions, if the PR is able to be broken down and these still hold true, then it should be broken down?

Reply:
I tend to agree with @kaspar030 on this. Maybe something like "Is the PR confined to a single explainable task?"

> Keeping PRs manageable is more of an incentive for authors: excessive contributions have a hard time to get reviewed.

I think this is only one side of the argument. For maintainers it is easier to spot flaws and judge impact with contributions changing a single task.